src/main/java/org/broadinstitute/consent/http/db/DataSetDAO.java

```diff
@@ -99,6 +99,13 @@
             "where d.dataSetId = :dataSetId order by d.dataSetId, k.displayOrder")
     Set<DataSetDTO> findDataSetWithPropertiesByDataSetId(@Bind("dataSetId") Integer dataSetId);
 
+  @UseRowMapper(DataSetPropertiesMapper.class)
+  @SqlQuery("select d.*, k.key, dp.propertyValue, ca.consentId, c.dac_id, c.translatedUseRestriction " +
+      "from dataset d left outer join datasetproperty dp on dp.dataSetId = d.dataSetId left outer join dictionary k on k.keyId = dp.propertyKey " +
+      "left outer join consentassociations ca on ca.dataSetId = d.dataSetId left outer join consents c on c.consentId = ca.consentId " +
+      "where d.dataSetId = :dataSetId order by d.dataSetId, k.displayOrder")
+  Set<DataSetDTO> findDataSetWithPropertiesByDataSetIdWithOuterJoins(@Bind("dataSetId") Integer dataSetId);
```

Comment:
nit, casing:
```suggestion
  Set<DataSetDTO> findDataSetWithPropertiesByDataSetIdWithOuterJoins(@Bind("datasetId") Integer datasetId);
```

Reply:
One more bit of naming feedback. Can we name this something like,
* findDatasetByIdWithProps
* findDatasetDTOById
Or something like that? Naming is always hard, but I'd have a hard time understanding what this really means without examining the sql, which can really slow a developer down when they're looking at lots of these in different daos.